### ISSUE TITLE
Unable to run kafka-minion with TLS_ENABLED=true

### ISSUE THREAD

**Issue by iMajna** [cloudworkz/kafka-minion#33]
Hello,

Tried to run a docker with KAFKA_TLS_ENABLED=true variable and container was unable to start. 

_env.list_
```
KAFKA_BROKERS=ip:9092
KAFKA_VERSION=2.1.0
KAFKA_TLS_ENABLED=true
KAFKA_TLS_CA_FILE=/etc/ssl/certs/ca-bundle.pem
KAFKA_TLS_KEY_FILE=/etc/ssl/certs/host.pem.key 
KAFKA_TLS_CERT_FILE_PATH=/etc/ssl/certs/host.pem
KAFKA_TLS_INSECURE_SKIP_TLS_VERIFY=false
KAFKA_TLS_PASSPHRASE=
```

_DockerFile_
```
# build image
FROM golang:1.13-alpine as builder
RUN apk update && apk add --no-cache git ca-certificates && update-ca-certificates

WORKDIR /app
COPY . .

RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -installsuffix cgo -o /go/bin/kafka-minion

# executable image
FROM alpine:3.11
COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
COPY --from=builder /go/bin/kafka-minion /go/bin/kafka-minion
COPY ./certs /etc/ssl/certs

ENV VERSION 1.0.0
ENTRYPOINT ["/go/bin/kafka-minion"]
```

**Command**
`docker run --network=host  --env-file env.list    kafka-minion
`
**ERROR Log**
```
{"level":"info","msg":"Starting kafka minion version1.0.0","time":"2020-02-20T14:23:33Z"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x98 pc=0x9325b1]

goroutine 1 [running]:
github.com/google-cloud-tools/kafka-minion/kafka.saramaClientConfig(0xc000178240, 0xc00011d980)
	/app/kafka/connection_helper.go:38 +0x151
github.com/google-cloud-tools/kafka-minion/kafka.NewCluster(0xc000178240, 0xc00005a3c0, 0xc00005a3c0)
	/app/kafka/cluster.go:74 +0x255
main.main()
```

**List dir**
```
docker exec -it 549e79f55ac4 sh -c "ls /etc/ssl/certs/ && echo 'It exists'"
ca-bundle.pem        host.pem.key
ca-certificates.crt  host.pem
It Exists
```

**Comment by mikekamornikov** [comment]
@iMajna Try `v1.0.0` (not `latest`).
@weeco This piece cause segfault as it doesn't init TLS config (`&tls.Config{`):
```
// kafka/connection_helper.go#L28-L32
	if opts.TLSEnabled {
		clientConfig.Net.TLS.Enable = true
		clientConfig.Net.TLS.Config.InsecureSkipVerify = opts.TLSInsecureSkipTLSVerify

		if opts.TLSCAFilePath != "" {
```

**Comment by mikekamornikov** [comment]
Additionally I have the same issue I had with `danielqsj/kafka_exporter`. With `MSK` `TLS` endpoints I have to set `KAFKA_TLS_INSECURE_SKIP_TLS_VERIFY` to true.

**Comment by weeco** [comment]
@mikekamornikov Thanks for pointing out the bug. I'll take care of it.

Regarding the insecure skip tls verify setting: If this is needed or not depends on your certificates. Do you use self signed certificates?

**Comment by mikekamornikov** [comment]
@weeco Nope, TLS piece is provider by amazon. I don't configure any certificate locations. In my case Kafka (AWS MSK) broker endpoints are TLS ones. And connection works just fine in another Golang consumer using the latest `sarama` with the following config:
```go
	config := sarama.NewConfig()
	config.ClientID = group
	config.Version = kafkaVersion // Kafka cluster version has to be defined before the consumer/producer is initialized
	config.Net.TLS.Enable = true  // enable SSL

	if oldest {
		config.Consumer.Offsets.Initial = sarama.OffsetOldest
	}
```
May it be related to fixed kafka version in `1.0.0`?

**Comment by adinhodovic** [comment]
@weeco Could we fix this in the helm chart too? Enabling tls

> @weeco Nope, TLS piece is provider by amazon. I don't configure any certificate locations. In my case Kafka (AWS MSK) broker endpoints are TLS ones. And connection works just fine in another Golang consumer using the latest `sarama` with the following config:
> 
> ```go
> 	config := sarama.NewConfig()
> 	config.ClientID = group
> 	config.Version = kafkaVersion // Kafka cluster version has to be defined before the consumer/producer is initialized
> 	config.Net.TLS.Enable = true  // enable SSL
> 
> 	if oldest {
> 		config.Consumer.Offsets.Initial = sarama.OffsetOldest
> 	}
> ```
> 
> May it be related to fixed kafka version in `1.0.0`?

@weeco Could we fix this in the helm chart too, not needing to configure TLS certs but using TLS due to AWS MSK endpoints having TLS?

**Comment by weeco** [comment]
@adinhodovic Do you mind submitting a PR for that in the chart repo?
